e2e/test/scenarios/models/create.cy.spec.js

```diff
@@ -26,63 +23,21 @@ describe("scenarios > models > create", () => {
     // Clicking on metadata should not work until we run a query
     cy.findByTestId("editor-tabs-metadata").should("be.disabled");
 
-    H.NativeEditor.focus().type("select * from ORDERS");
-
+    H.NativeEditor.focus().type("select 42");
     cy.findByTestId("native-query-editor-container").icon("play").click();
     cy.wait("@dataset");
 
     cy.findByTestId("dataset-edit-bar").button("Save").click();
-    cy.findByPlaceholderText("What is the name of your model?").type(modelName);
-
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Save").click();
-
-    // After saving, we land on view mode for the model
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.findByText("Summarize");
-
-    checkIfPinned();
-  });
-
-  it("suggest the currently viewed collection when saving a new native query", () => {
```

Comment:
There's no such option in any of the remaining UI triggers anymore.
@npfitz maybe you'd want to the collection-id context one to the new model in command palette?

This was the previous implementation
https://github.com/metabase/metabase/pull/57647/files#diff-ac19de8dbea1999bb08ec6b9e13d5e2451f39f061503809a26c075f2feef086dL113